.drone-v1.yml

```
@@ -0,0 +1,268 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: install dependancies
```

```suggestion
  - name: install dependencies
```

A dependant is a person.

---

Done 👍🏼